Lesson 18/task-1.py

```diff
@@ -0,0 +1,34 @@
+import re
+
+
+VALID_EMAILS = ['abc-d@mail.com', 'abc.def@mail.com', 'abc@mail.com', 'abc_def@mail.com',
+                'abc.def@mail.cc', 'abc.def@mail-archive.com', 'abc.def@mail.org', 'abc.def@mail.com']
+
+INVALID_EMAILS = ['abc-@mail.com',  'abc..def@mail.com', '.abc@mail.com', 'abc#def@mail.com',
+                  'abc.def@mail.c', 'abc.def@mail#archive.com', 'abc.def@mail', 'abc.def@mail..com']
+
+
+class CreateEmail:
+    def __init__(self, email):
+        self.email = email
+        self.is_valid = self.validate(email)
+
+    @staticmethod
+    def validate(email):
+        regex = r'([A-Za-z0-9]+[-._])*[A-Za-z0-9]+@[A-Za-z0-9-]+(\.[A-Z|a-z]{2,})'
+
+        if re.fullmatch(regex, email):
```

Comment:
I would do it like this:
`
is_valid = re.fullmatch(regex, email)
is_valid_to_str = 'valid' if is_valid else 'invalid'
print(f'{is_valid_to_str} - {email}')
return is_valid
`